.gitignore

```diff
@@ -2,3 +2,4 @@ Containerfile
 .tox
 __pycache__
 *.pyc
+.idea/
```

Comment:
We probably shouldn't push every single custom things. We should rely on the global ignore capability provided by git, as it's done with upstream Openstack repositories:
```
git config --global core.excludesfile "~/.gitignore"
```
and in ~/.gitignore, you add whatever you want.

Reply:
Fixed.